mods/webui/src/stories/overviewcard/OverviewCard.stories.ts

```diff
@@ -77,7 +79,11 @@ export const Example: Story = {
 export const ChangeIcon: Story = {
   args: {
     label: "Workspace setting",
-    icon: React.createElement(Icon, { name: 'Search', fontSize: "small" })
+    icon: React.createElement(Icon, {
+      name: "Search",
+      fontSize: "small",
+      sx: { color: "#008751" }
```

Comment:
It feels like this (the color) could be part of the component itself.

Thoughts?

Reply:
hmmm, Here the icon props takes in the component that requires the color
I will attempt adding the color to the parent component to see if that works as well

Reply:
@psanders 
It worked. 🔥